build.rs

```diff
@@ -288,10 +288,10 @@ fn configure() {
     let version = std::fs::read_to_string("version.txt").unwrap();
     let version = version.trim();
     let version_parts: Vec<&str> = version.split('.').collect();
-    let major = version_parts[0];
-    let minor = version_parts[1];
-    let patch = version_parts[2];
-    let tweak = version_parts[3];
+    let major = version_parts.first().unwrap_or(&"");
```

Comment:
I believe major, minor, and patch are always required, isn't it?
IMHO only the tweak parameter is optional.

Reply:
Fixed